mealie/schema/response/pagination.py

```diff
@@ -23,6 +23,7 @@ class RecipeSearchQuery(MealieModel):
     require_all_tools: bool = False
     require_all_foods: bool = False
     search: str | None
+    timestamp: float | None
```

Comment:
I think using a string and defaulting to empty would help avoid some possible null errors. Maybe be worth renaming this to `randomSeed` for clarity as well

```suggestion
    timestamp: string = ""
```

Reply:
Performance is great on my 6,000 recipe database. I don't notice any difference between sorting by Category or Random. This is with a less-than-fast backend (2nd gen intel Core CPU). Grabbing just the recipe ids takes advantage of the index, which really speeds things up. And after that, it's just a python list shuffle. Which might be slow for a million-recipe database, but not for thousands.